dataprofiler/profilers/profile_builder.py

```diff
@@ -2290,7 +2293,11 @@ def _update_null_replication_metrics(self, clean_samples: Dict) -> None:
             sum_not_null = np.delete(total_row_sum, col_id) - sum_null
 
             mean_null = sum_null / null_count
-            mean_not_null = sum_not_null / true_count
+
+            if true_count == 0:
+                mean_not_null = sum_not_null
+            else:
+                mean_not_null = sum_not_null / true_count
```

Comment:
is this necessary?

Reply:
When a column is all null, `true_count` is 0, which creates a divide by 0 error.